### ISSUE TITLE
fix: `useLocalePath` has no type access to `route`

### ISSUE THREAD

**Issue by dargmuesli** [victorgarciaesgi/nuxt-typed-router#130]
In

https://github.com/victorgarciaesgi/nuxt-typed-router/blob/5353bb7c34d9440ef1f2f9a257620f30d37d2243/src/core/output/generators/files/__i18n-router.file.ts#L51

why pick `'i18n'` only?

In a middleware I want to use
```
defineNuxtRouteMiddleware(async (to) => {
      const localePath = useLocalePath({ route: to })
})
```
but am currently not able to, at least type wise.

I'd propose to add `| 'route'`, or even drop the pick.

**Comment by victorgarciaesgi** [comment]
There should have been a reason because of wrong types on `@nuxtjs/i18n` side. 
(It was in beta so things where missing)
I look at it maybe it's not the case anymore

**Comment by victorgarciaesgi** [comment]
Fixed in `v3.5.0`
